analysis/TBD/TBD.go

```diff
@@ -62,9 +62,9 @@ func BuildTBDCollection(res *database.Resources) {
 	newTBD(res).run()
 }
 
-func GetTBDResultsView(res *database.Resources, cutoffScore float64) *mgo.Iter {
+func GetTBDResultsView(res *database.Resources, ssn *mgo.Session, cutoffScore float64) *mgo.Iter {
```

Comment:
Functions which aggregate and return an iterator need a session to use. Before this change, we created a session copy inside the method, closed it, and returned the iterator out. A call to next on the iterator would break the program.